docs/setup-guide/multi-node-dev-cluster.md

```diff
@@ -1,52 +1,73 @@
 # How to Setup a small Dev Cluster of multi-nodes
 
-It may be desired to setup a dev cluster having 2 or 3 nodes in order to play with comprehansive features of Arktos. The simple and easy-to-use arktos-up.sh feels short in this case; this doc describes the minimum effort to run a cluster havinbg 2 nodes (1 master, plus a worker who joins later).
+It may be desired to setup a dev cluster having more worker nodes in order to play with comprehensive features of Arktos. The simple and easy-to-use arktos-up.sh feels short in this case; this doc describes the minimum effort to run a cluster having 1 master node and extra worker nodes joining after the master is up.
 
-This doc, as at the moment written, does not mandate which cni plugin to be used. It is up to user, though we recommend to use calico, or our own alktron.
+This doc, as at the moment written, does not mandate which cni plugin to be used; it is up to the user. We have verified that Flannel works well with multi-tenancy Arktos cluster; the instructions laid out here is based on our experience with Flannel at GCP env.
 
-1. start the master node 
- the master node, run follwoing commands to start cluster, which has the master only.
+1. bootstrap the cluster by starting the master node (no CNI plgin at first)
```

Comment:
First tell the user to cd to the repo root.

Reply:
will state this assumption explicitly.